help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,116 @@
+
+## Use cases 
+
+To help you better understand how and when you should use the Customer Decision Hub destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1
+
+*In Telco:*
+
+*A marketer wants to leverage insights from Data Science model-based next best action as delivered by Pega Customer Decision Hub for customer engagement, Pega Customer Decision Hub is heavily reliant on customer intent – for example "Interested_In_5G", "Interested_in_Unlimited_Dataplan" or "Interest_in_iPhone_accessories".*
+
+### Use case #2
+
+*In Financial Services:*
+
+*A marketer wants to optimize the offers for customers who subscribed or unsubscribed for Pension Plan or Retirement Plan newsletters. The financial services can ingest multiple CustomerID from their own CRM to Adobe Experience Platform, build segments from their own offline data, and send profiles that are entering and exiting the segments to Pega Customer Decision Hub for next-best-action (NBA) decisioning in outbound channels.*
+
+## Prerequisites {#prerequisites}
+
+Before you use Pega Customer Decision Hub destination to export data out of Adobe Experience Platform:
+
+* Configure the Adobe Segment Membership Component in your Pega Customer Decision Hub instance.
+* Configure OAuth 2.0 Client Registration using Client Credentials grant type in your Pega Customer Decision Hub instance
+* Configure real-time processing of Adobe Segment Membership data flow to run in your Pega Customer Decision Hub instance.
+
+## Supported identities {#supported-identities}
+
+Pega Customer Decision Hub supports the activation of custom user ID described in the table below. For more details, see [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description
+|---|---|
+|*CustomerID*|Common User Identifier that uniquely identifies a profile in Pega Customer Decision Hub and Adobe Experience Platform
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | Export all members of a segment with identifier (*CustomerID*), attributes (last name, first name, location,..) and Segment Membership data.|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are an always on API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. For more information, see [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+Follow the steps described  to connect to this destination. In the configure destination workflow, fill in the fields listed in the two sections below.
```

Comment:
Minor comment, the boilerplate text was linking to a generic tutorial about how to connect to destinations. Let's keep it, please: 

To connect to this destination, follow the steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html). In the configure destination workflow, fill in the fields listed in the two sections below.